README.md

```diff
@@ -150,6 +154,57 @@ module "my-ez-fargate-service" {
 }
 ```
 
+### Internal Example
+
+An example demonstrating that if only private subnets are given then the ALB will be automatically configured as internal.
+
+```terraform
+module "my-ez-fargate-service" {
+  source             = "USSBA/easy-fargate-service/aws"
+  version            = "~> 4.0"
+  family             = "my-ez-fargate-service"
+  container_image    = "nginx:latest"
+  cluster_name       = "my-ecs-cluster"
+  desired_capacity   = 2
+  max_capacity       = 4
+  min_capacity       = 2
+  scaling_metric     = "cpu"
+  scaling_threshold  = 75
+  vpc_id             = "vpc-1234abcd"
+  private_subnet_ids = ["subnet-11111111", "subnet-22222222", "subnet-33333333"]
+  hosted_zone_id     = "Z000000000000"
+  service_fqdn       = "www.cheeseburger.com"
+}
+```
+
+### Listener Configuration Example
+
+An example demonstrating how to override listener configuraiton. If a certificate is not provided then the HTTPS forward listener will not be provisioned.
```

Comment:
Probably not necessary to have examples in the readme. Just have links to the examples directory